data/json/monsters/civilians.json

```diff
@@ -167,7 +167,7 @@
   {
     "id": "mon_civilian_zombiegunner",
     "type": "MONSTER",
-    "name": { "str": "firearmed fighter" },
+    "name": { "str": "armed fighter" },
```

Comment:
I believe the point is that they have a firearm, as opposed to being armed in some other way.

Reply:
Even then, "firearmed" is not a fitting descriptor. Nor is it a descriptor at all. It's not a real word.

Reply:
Armed is often used as a way to say someone is armed with a firearm

Reply:
@Holli-Git You can be armed with a sword, for instance. I agree that firearmed is not a word, but we need to find something else that conveys "this guy has a gun"

Reply:
Do we though? I think for the majority of people, they're going to see "armed" and assume it is with a gun. Also, the mob is likely to be removed anyways, so I don't feel like putting in effort to change it